bunq/sdk/context.py

```diff
@@ -254,45 +254,55 @@ def session_context(self):
 
         return self._session_context
 
-    def save(self, path=None, to_json=False):
+    def to_json(self):
+        """
+        Serializes an ApiInstance to JSON data
+
+        :return: str
+        """
+        return converter.class_to_json(self)
+
+    @classmethod
+    def from_json(cls, data):
+        """
+        Creates an ApiContext instance from JSON data
+
+        :param data: str
+        :return: ApiContext
```

Comment:
Shouldn't this be `:rtype:`? And could you place a new line before `:type:` & `:return:` please.